src/securejoin.rs

```diff
@@ -1424,6 +1231,35 @@ mod tests {
             "vg-member-added"
         );
 
+        {
+            // TODO: Check if Alice has a hidden 1:1 chat in which there is an info message
+            // saying bob is verified.  However currently this is purposefully sent to
+            // Alice's group chat by commit 3b7b8ea0f1f8b9c9d300388768eed069be59c8be?
```

Comment:
@r10s would you like to check which way is really desired here?  I think the current implementation shows the "contact verified" message in a group chat for alice.  So if she invites 5 people to a group with QR code she'd get 5 verified messages in the group while the 1:1 chat with those 5 people is still hidden and empty.  Was that the intention?

Reply:
And likewise, how should it behave for bob?

Reply:
i think, this is fine.

the intention  for both, Alice and Bob, is to avoid creation of currently unneeded chats.

since https://github.com/deltachat/deltachat-core-rust/pull/2508, for Bob, directly after scanning the group chat is opened and in focus of the user. it makes sense to show all messages there. creating a second one-to-one chat when scanning a group is confusing an not needed.

for Alice, it makes sense to have a similar behavior; also as otherwise Alice's chat list is easily spammed with one-to-one chats of all joiners.

Reply:
Finally got round to fixing up this bit.  I'll rebase and merge later this week.